docs/faqs.md

```diff
@@ -65,11 +67,12 @@ Boom, you have a new follower!
 If you ever want to add Listings locally, you must have credits on your account
 to "pay" for listing. Here's how:
 
-1. Open the rails console `rails c`.
+1. Open the rails console `rails console`.
 2. Enter the following commands:
+
    ```ruby
-   user = User.find_by(username: "your_username")
-   Credit.add_to(user, 1000)
+   user = User.find_by(username: "your_username");
```

Comment:
Prettier was over-eager here, there shouldn't be semi-colons in Ruby. Whatever the problem was, this is not the fix, semicolons in Pry just surpress showing output.

Reply:
Sounds good. Thanks. I added the `;`, so that's on me. 😄

Reply:
@citizen428, the funny thing is now that I removed the semi-colons, it appears to be just prettier formatting in action. I wonder if the empty line between the code snippet and the text was what fixed this.